tools/cli/go-whisk-cli/commands/trigger.go

```diff
@@ -415,26 +421,30 @@ var triggerDeleteCmd = &cobra.Command{
 
```

Comment:
That means no feed is associated with the trigger, so deletion of the feed is not carried out.